examples/facebook/package.json

```diff
@@ -30,7 +30,7 @@
     "url-loader": "^4.1.1"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.15.7",
+    "@babel/eslint-parser": "^7.16.3",
```

Comment:
while you are at it, can you align "prettier": across packages?

"prettier": "^2.5.0",

```
packages/create-docusaurus/templates/facebook/package.json 
```

Reply:
I submitted this PR right before I went out and I planned to merge it on the phone 😄

Still, good catch, I guess I'll do a more serious dependency cleanup when I get back on my computer